### ISSUE TITLE
KaOS 2016.11 fails to install

### ISSUE THREAD

**Issue by adomas187** [KaOSx/calamares#73]
#### Submission type

  - [x] Bug report
  - [ ] Feature Request

#### KaOS ISO date you are having issues with or have a request for

> KaOS 2016.11

#### How did you create the Live Media?

> Using Rufus on Windows 10

#### Did you check the md5sum of the downloaded ISO?

> No

#### Describe the issue you encountered

> Every time I try to install KaOS 2016.11 using Calamares installed at 19% I get this error message:

Boost.Python error in job "userkf5".
<class 'AttributeError'>
'NoneType' object has no attribute 'group'
Traceback:
  File "/usr/lib/calamares/modules/userkf5/main.py", line 115, in run
    print (m.group(0))

#### Steps to reproduce the problem

> Happens every time I try to install

#### Include the installation.log:
 * Live mode: `~/installation.log`
 * Installed System: `/var/log/installation.log`

> [installation.log.txt](https://github.com/KaOSx/calamares/files/587535/installation.log.txt)

**Comment by demmm** [comment]
How was the live media created?  See the warnings in the News, Release Notes and links for how to create media what NOT to use.

**Comment by adomas187** [comment]
Well I have experience with Rufus working on unsupported ISO files. I think this is not Rufus problem this time. Had a small issue with booting live media, but solved like always (happens with every unsupported distro).

Simple solution to support Rufus would be: Cut the ISO label, since it is always shortened in /dev/disk/by-label/

At current state: booting from USB made with Rufus do these steps:
When thrown into command line, write: ln -s /dev/disk/by-label/KAOS_201611 /dev/disk/by-label/KAOS_20161104
then type: exit
Live medium boots as expected.

Back on topic, will try dd iso on my other machine right now.

**Comment by demmm** [comment]
As noted all over the KaOS website, Rufus is NOT supported and always fails the install, mostly with the exact error you have here.
Closing this report, warnings have been posted plenty.  Not a valid report.

**Comment by adomas187** [comment]
Options mentioned in KaOS website have failed to burn ISO to my USB media. Some kind of access errors in graphical programs, and simple crash with dd.exe

I have managed to burn ISO with Win32 Disk Imager. Live medium booted smoothly, started Calamares, and after 96% I get this error message:
<class 'FileNotFoundError'>

No such file or directory: '/tmp/calamares-root-8nvely6u/boot/loader/entries/KaOS 2016.01.conf'

Traceback:

File "/usr/lib/calamares/modules/bootldr/main.py", line 164, in run

install_bootloader(boot_loader, fw_type)

File "/usr/lib/calamares/modules/bootldr/main.py", line 146, in install_bootloader

create_conf(uuid, conf_path)

File "/usr/lib/calamares/modules/bootldr/main.py", line 71, in create_conf

with open(conf_path, 'w') a

Seems same error had this guy on forum:
https://forum.kaosx.us/d/1381-installation-problem

As demm wrote I should chose EFI partition, but my Calamares does not let me choose, only /dev/sda2 as EFI is selected (in disk partitioning I chose replace partition).

**Comment by adomas187** [comment]
It seems that really was Rufus problem. For me dd.exe crashed, Rosa ImageWrite and Suse Studio ImageWriter crashed with some kind of access error, and only Win32 Disk Imager helped to prepare USB live medium.

After booting up USB medium, I got the same problem as user ghostmm on KaOS forum:
https://forum.kaosx.us/d/1381-installation-problem
Since I have newer version of KaOS with more recent Calamares, somehow could not choose EFI partition in Disk Partitioning, but was forced to use EFI partition /dev/sda2. So the demm solution to that topic did not help. Tried restarting Calamares and then installation seem to be finished without any errors. Rebooted the machine, and got into my new KaOS enviroment.

Now I am very happy. Thank you. Sorry for false bug report.
